doc/1/guides/kuzzle-depth/scalability/index.md

```diff
@@ -58,6 +58,13 @@ docker-compose -p cluster up --scale kuzzle=3
 
 You should now have a Kuzzle cluster stack running with 3 Kuzzle nodes.
 
+::: info
+<SinceBadge version="1.9.3" />
```

Comment:
```suggestion
<SinceBadge version="1.10.0" />
```
Since we are gonna release the new admin controller actions to encrypt/decrypt secrets